native/packages/react-native-bpk-component-button-link/readme.md

```diff
@@ -53,11 +51,6 @@ export default class App extends Component {
           icon="baggage"
           iconAlignment="leading"
           onPress={() => {}} />
-        <BpkButtonLink
-          title="Book flight"
-          icon="baggage"
-          iconOnly
-          onPress={() => {}} />
```

Comment:
Removed this example, as this component doesn't currently support `iconOnly` (was probably copypasta'd from `BpkButton`).